libs/application/templates/passport/src/lib/PassportTemplate.ts

```diff
@@ -6,84 +6,109 @@ import {
   ApplicationStateSchema,
   Application,
   DefaultStateLifeCycle,
+  DefaultEvents,
 } from '@island.is/application/core'
-import * as z from 'zod'
-
-const nationalIdRegex = /([0-9]){6}-?([0-9]){4}/
-
-type Events =
-  | { type: 'APPROVE' }
-  | { type: 'REJECT' }
-  | { type: 'SUBMIT' }
-  | { type: 'ABORT' }
-
-const dataSchema = z.object({
-  personalInfo: z.object({
-    name: z.string().nonempty(),
-    nationalId: z.string().refine((x) => (x ? nationalIdRegex.test(x) : false)),
-    phoneNumber: z.string().min(7),
-    email: z.string().email().nonempty(),
-    otherEmail: z.string().email().nonempty(),
-    height: z.string().nonempty(),
-  }),
-  service: z.object({
-    type: z.enum(['regular', 'express']),
-    dropLocation: z.enum(['1', '2', '3']),
-    extraOptions: z
-      .array(z.union([z.enum(['bringOwnPhoto']), z.undefined()]))
-      .nonempty(),
-  }),
-  fetchData: z.boolean().refine((v) => v),
-})
+import { dataSchema } from './dataSchema'
+import { Roles, States, Events, ApiActions } from './constants'
+import { Features } from '@island.is/feature-flags'
+import { m } from '../lib/messages'
 
 const PassportTemplate: ApplicationTemplate<
   ApplicationContext,
   ApplicationStateSchema<Events>,
   Events
 > = {
   type: ApplicationTypes.PASSPORT,
-  name: 'Umsókn um vegabréf',
+  name: m.formName.defaultMessage,
+  featureFlag: Features.passportApplication,
   dataSchema,
   stateMachineConfig: {
-    initial: 'draft',
+    initial: States.DRAFT,
     states: {
       draft: {
         meta: {
-          name: 'Umsókn um vegabréf',
+          name: m.formName.defaultMessage,
           progress: 0.33,
           lifecycle: DefaultStateLifeCycle,
```

Comment:
All the lifecycle settings are on default lifecycle which means the data persists forever. I recommend you update those if they are not correct.